CONTRIBUTING.md

```diff
@@ -116,6 +116,39 @@ Once the changes are merged on `main`, you can publish the current draft release
 
 GitHub Actions will be triggered and push the package to [npm](https://www.npmjs.com/package/meilisearch).
 
+#### Release a `beta` Version
+
+Here are the steps to release a beta version of this package:
+
+- Create a new branch originating the branch containing the "beta" changes. For example, if during the MeiliSearch pre-release, create a branch originating `bump-meilisearch-v*.*.*`.<br>
+`vX.X.X` is the next version of the package, NOT the version of MeiliSearch!
+
+```bash
+git checkout bump-meilisearch-v*.*.*
+git pull origin bump-meilisearch-v*.*.*
+git checkout -b vX.X.X-beta.0
+```
+
+- Change the version in `package.json` by `vX.X.X-beta.0` and commit it to the `vX.X.X-beta.0` branch
```

Comment:
```suggestion
- Change the version in `package.json` by `vX.X.X-beta.0` and commit it to the `vX.X.X-beta.0` branch
```
It is a little detail but I did not know how to name my commit
` Create beta version v0.X.X.X-beta.0` ?
Should we add it?

Reply:
No, the name of the commit does not impact any other part of the beta release process. Put the name you want, following our commit conventions :)